fdbserver/LogSystem.h

```diff
@@ -54,6 +56,8 @@ struct ConnectionResetInfo : public ReferenceCounted<ConnectionResetInfo> {
 class LogSet : NonCopyable, public ReferenceCounted<LogSet> {
 public:
 	std::vector<Reference<AsyncVar<OptionalInterface<TLogInterface>>>> logServers;
+	// question: why above we use reference -> asyncvar -> optionalinterface? Can I skip them here?
```

Comment:
what's wrong with reference -> asyncvar ?

Reply:
Oh I can get the interface directly from `state ptxn::TLogInterface_PassivelyPull serverNew = wait(reqs[i].ptxnReply.getFuture());` , I am not sure why we use it this way,